e2e/epinio.e2e.spec.ts

```diff
@@ -0,0 +1,238 @@
+   */
+  test.afterAll(tearDownEpinio);
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: playwrightReportAssets(path.basename(__filename)) });
+    await electronApp.close();
+  });
+
+  test('should start loading the background services', async() => {
+    const navPage = new NavPage(page);
+
+    await navPage.progressBecomesReady();
+    await expect(navPage.progressBar).toBeHidden();
+  });
+  test('should check kubernetes API is ready', async() => {
+    const output = await kubectl('cluster-info');
+
+    expect(output).toMatch(/is running at ./);
+  });
+  test('should verify epinio cli was properly installed', async() => {
+    const epinioCliStatus = await epinio('version');
+
+    expect(epinioCliStatus).toContain('Epinio Version');
+  });
+  test('should add epinio-installer helm repository', async() => {
+    const epinioRepoAdd = await helm('repo', 'add', 'epinio', 'https://epinio.github.io/helm-charts');
+
+    expect(epinioRepoAdd).toContain('"epinio" has been added to your repositories');
+  });
+  test('should install epinio-installer application', async() => {
+    const loadBalancerIpAddr = await loadBalancerIp();
+    const epinioInstall = await helm('install', 'epinio-installer', 'epinio/epinio-installer',
+      '--set', 'skipTraefik=true', '--set', `domain=${ loadBalancerIpAddr }.omg.howdoi.website`,
+      '--wait', '--timeout=25m');
+
+    expect(epinioInstall).toContain('STATUS: deployed');
+  });
+  test('should update epinio config certs file', async() => {
+    const epinioConfigUpdate = await epinio('config', 'update');
+
+    expect(epinioConfigUpdate).toContain('Ok');
+  });
+  test('should push a sample app through epinio cli', async() => {
+    const epinioPush = await epinio('push', '--name', 'sample', '--path', path.join(__dirname, 'assets', 'sample-app'));
+
+    expect(epinioPush).toContain('App is online.');
+  });
+  test('should verify deployed sample application is reachable', async() => {
+    const loadBalancerIpAddr = await loadBalancerIp();
+    const urlAddr = `https://sample.${ loadBalancerIpAddr }.omg.howdoi.website`;
+    // Trick to avoid error 60 (SSL Cert error), passing "--insecure" parameter
+    const sampleApp = await curl('--fail', '--insecure', urlAddr);
+
+    expect(sampleApp).toContain('PHP Version');
+  });
+});
+
+/**
+ * Helper to identify the Load Balancer IP Address.
+ * It will return the traefik IP address, required by epinio install.
+ */
+export async function loadBalancerIp() {
+  const serviceInfo = await kubectl('describe', 'service', 'traefik', '--namespace', 'kube-system');
+
+  const serviceFiltered = serviceInfo.split('\n').toString();
+  const ipAddrRegex = /(LoadBalancer Ingress:)\s+(((?:[0-9]{1,3}\.){3}[0-9]{1,3}))/;
+  const regex = new RegExp(`${ ipAddrRegex.source }`);
+  const ipAddressLb = regex.exec(serviceFiltered);
+
+  // checking if it will be undefined, null, 0 or empty
+  if (typeof ipAddressLb !== 'undefined' && ipAddressLb) {
```

Comment:
```suggestion
  if (ipAddressLb) {
```
If it's undefined, it's falsy.

Reply:
Done.